src/utilities/content-tree-enhancers.js

```diff
@@ -5,6 +5,7 @@ const remark = require('remark');
 const slug = require('remark-slug');
 const extractAnchors = require('remark-extract-anchors');
 const remarkHtml = require('remark-html');
+const frontmatter = require('remark-frontmatter');
```

Comment:
What changed that this is now necessary?

Reply:
After upgrading remark to 13, there seems to be a bug in `remark-extract-anchors` when processing this article https://github.com/webpack/webpack.js.org/blob/master/src/content/blog/2020-10-10-webpack-5-release.md. It would parse the front matter as heading:

![image](https://user-images.githubusercontent.com/1091472/98111625-d9761e00-1edb-11eb-8e65-b86709e03305.png)

![image](https://user-images.githubusercontent.com/1091472/98111694-f579bf80-1edb-11eb-8ac1-3357e9fa72c6.png)

Which is wrong, hence I include `remark-frontmatter` to process the front matter first.

Reply:
Ok, I found it can be easily fixed by deleting indentation from the front matter.

Before:

```yaml
---
title: Webpack 5 release (2020-10-10)
sort: -202010100
contributors:
    - sokra
    - chenxsan
---
```
After:

```yaml
---
title: Webpack 5 release (2020-10-10)
sort: -202010100
contributors:
  - sokra
  - chenxsan
---
```
Anyway, I believe including a remark plugin is much safer here as it can prevent potential issues in future.